CHANGES.rst

```diff
@@ -1,22 +1,22 @@
 Changes
 =======
 
-Version master (UNRELEASED)
----------------------------
-
-- Adds option to filter job logs according to compute backend, docker image, status and steps.
-- Enriches disk usage output format.
-- Adds new command to restart workflows.
-- Optimizes CLI performance.
-- Adds option to get logs of a specific step.
-- Enhances logs formatting using marks and colors.
-- Switches from Bravado to ``requests`` to perform better performant downloads.
-- Validates user access token and improves output of ``ping`` command.
-- Accepts both ``reana.yaml`` and ``reana.yml`` filenames.
-- Includes better formatting for ``diff`` command.
-- Prevents including a dot as part of the workflow name.
-- Allows passing workflow operational options in the ``reana.yaml``.
-- Adds Black formatter support.
+Version 0.7.0 (2020-10-19)
+--------------------------
+
+- Adds option to ``logs`` command to filter job logs according to compute backend, docker image, status and step name.
+- Adds new ``restart`` command to restart previously run or failed workflows.
+- Adds option to specify operational options in the ``reana.yaml`` of the workflow.
```

Comment:
What do you think about "Adds possibility..." or similar to avoid the "option" repetition?